### PR TITLE
DOCSP-13323: Realm Object Queries

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-13323

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-13323/android/reads#find-an-object-by-primary-key
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-13323/android/reads#query-a-relationship
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-13323/android/reads#query-an-inverse-relationship

### CODE REVIEW

**Review by nlarew — CHANGES_REQUESTED** [review]
A few nits/questions + a couple other things:
- **Main change** - Let's move all the examples to the **Reads** page rather than have them on their concept pages. In the New Realm IA they're all included on the **Read & Write Data** page so it's easier if we centralize them now.
- Thoughts on omitting the imports from the code blocks? I'm not strongly opinionated here, but it seems like they might distract from the examples. Do you think they're worthwhile to keep for the sake of runnability?

File: source/android/objects.txt [hunk 1]
```diff
@@ -751,6 +751,27 @@ your {+realm+} using that module:
       .. literalinclude:: /examples/Schemas/Modules.java
          :language: java
 
+Find an Object by Primary Key
+-----------------------------
+
+To find an object with a specific primary key value, open a {+realm+}
+and query the primary key field for the desired primary key value
+using the :java-sdk:`RealmQuery.equalTo() <TODO>` method:
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/android/objects.txt:759]: unfortunately i don't think there's a shortcut method

[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/android/objects.txt:759]: Just double-checked: no shortcut. But I did fill in the TODO link 😉

File: source/android/relationships.txt [hunk 2]
```diff
@@ -157,6 +157,92 @@ set the value of an inverse relationship property.
...
+      of the linked object:
+
+      .. literalinclude:: /examples/generated/android/code/start/RealmQueryTest.codeblock.query-a-relationship.kt
+         :language: kotlin
+         :emphasize-lines: 15
+
+   .. tab::
+      :tabid: java
+
+      Consider the following relationship between classes ``Human`` and ``Cat``:
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/android/relationships.txt:187]: I'd like to show both the class declarations and the code that queries those classes, and without some seriously annoying hides/replaces, I can't use the same names twice in the same example project. I figured this was the best possible alternative without serious hoop jumping, though I would prefer being able to use the same names across examples.

File: source/android/relationships.txt [hunk 3]
```diff
@@ -157,6 +157,92 @@ set the value of an inverse relationship property.
...
 
+Query a Relationship
+--------------------
+
+.. tabs-realm-languages::
+
+   .. tab::
+      :tabid: kotlin
+
+      Consider the following relationship between classes ``Person`` and ``Dog``:
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/android/relationships.txt:168]: Going to add an emphasize-lines to the RealmObject field as well, thanks!

**Comment by nathan-contino-mongo** [comment]
Going to keep the imports for the sake of runnability and code-completeness entirely because my personal preference is to always have imports in documentation. It would be nice if we had some way to accordion them away or de-emphasize them, but I think it's fine this way.

**Review by nlarew — APPROVED** [review]

**Pull request merged**